ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/_cq_dialog/.content.xml

```diff
@@ -79,14 +79,16 @@
                                                       sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
                                                       granite:class="cmp-adaptiveform-base__enum"
                                                       name="./enum"
-                                                      fieldLabel="Data Value"
+                                                      fieldLabel="Submit value"
```

Comment:
Submit Value does not looks much correct to me.

Reply:
can we call it `option title` and `option value` ?

Reply:
`Data Value` seems a standard term already. why use anything different? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select